dotcom-rendering/src/web/layouts/NewslettersPageLayout.tsx

```diff
@@ -0,0 +1,174 @@
+import { css } from '@emotion/react';
+import {
+	brandBackground,
+	brandBorder,
+	brandLine,
+	palette,
+} from '@guardian/source-foundations';
+import { StraightLines } from '@guardian/source-react-components-development-kitchen';
+import type { NavType } from '../../model/extract-nav';
+import type { DCRNewslettersPageType } from '../../types/newslettersPage';
+import { Footer } from '../components/Footer';
+import { Header } from '../components/Header';
+import { HeaderAdSlot } from '../components/HeaderAdSlot';
+import { Island } from '../components/Island';
+import { Nav } from '../components/Nav/Nav';
+import { NewslettersList } from '../components/NewsletterList';
+import { NewslettersPageHeading } from '../components/NewsletterPageHeading';
+import { Section } from '../components/Section';
+import { SubNav } from '../components/SubNav.importable';
+import { decideFormat } from '../lib/decideFormat';
+import { Stuck } from './lib/stickiness';
+
+type Props = {
+	newslettersPage: DCRNewslettersPageType;
+	NAV: NavType;
+};
+
+export const NewslettersPageLayout = ({ newslettersPage, NAV }: Props) => {
+	const {
+		subscribeUrl,
+		editionId,
+		pageFooter,
+		format,
+		contributionsServiceUrl: pageContributionsServiceUrl,
+		config,
+		isAdFreeUser,
+	} = newslettersPage;
+	const articleFormat: ArticleFormat = decideFormat(format ?? {});
```

Comment:
Sounds good - I was only adding a `format` so those components could be rendered. `decideFormat` defaults to 'news', but it would be better for this page to have a  `Nav` without a selected pillar.

Could we defer refactoring the "shared" components to the next PR so this one doesn't get too big?